### PR TITLE
Modal Filter Update

### PR DESCRIPTION
**Pull request by Joeedwin073**

### CODE REVIEW

File: app/Http/Controllers/ExpenseController.php [hunk 1]
```diff
@@ -24,6 +25,48 @@ public function ministry()
     }
```
Comment (goodhands) [inline app/Http/Controllers/ExpenseController.php:25]: Look at this. You have closed the class definition before defining your methods. Do you test your code locally at all?

File: resources/views/pages/expense/ministry.blade.php [hunk 2]
```diff
@@ -12,8 +13,18 @@
 @endpush
 
 @section('content')
-{{ Breadcrumbs::render('expense.ministry') }}
```
Comment (goodhands) [inline resources/views/pages/expense/ministry.blade.php:15]: Why did you remove this?

File: resources/views/pages/expense/ministry.blade.php [hunk 3]
```diff
@@ -44,105 +55,79 @@
...
-										@endforeach
-									</tbody>
-								</table>
-							</div>
+							@include('pages.expense.ministry_table')
 						</div>
+						
 						<!---PAGINATION--->
-						@include('partials.pagination', ['data' => $collection['summary']])
+						{{-- @include('partials.pagination', ['data' => $collection['summary']]) --}}
```
Comment (goodhands) [inline resources/views/pages/expense/ministry.blade.php]: Why are you commenting the pagination?

File: resources/views/pages/expense/Ministry_table.blade.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                @endforeach
+            </tbody>
+        </table>
+    </div>
+    <div class="table-footer d-flex align-items-center pagination1">
+        <span>{{ $collection['summary']->firstItem() }} - {{ $collection['summary']->lastItem() }} of {{ $collection['summary']->total() }} results</span>
+        {{ $collection['summary']->links() }}
+    </div>
+
+{{--
```
Comment (goodhands) [inline resources/views/pages/expense/Ministry_table.blade.php]: Why did you comment this out as well?

**Review by goodhands — CHANGES_REQUESTED** [review]
I see you are moving the ministry table into a separate file and including it. The right way to do that is to move it to the `partials` folder and not the `pages` folder. 

Secondly, you have to make sure everything works as expected after doing that. And I don't understand why you are removing the pagination but you have to leave that as it was. And breadcrumb as well

File: app/Http/Controllers/ExpenseController.php (hunk 1 above)
Comment (Joeedwin073) [reply app/Http/Controllers/ExpenseController.php:25]: No sir... It's no closed.
Kindly check it again
